README.md

```diff
@@ -1,4 +1,4 @@
-allthekeeps
+AllTheKeeps on Ropsten
```

Comment:
Can we describe what a developer has to configure and run to set up and run allthekeeps locally?

Reply:
Sure, but if we deal with local development improvements in a [separate PR](https://github.com/keep-network/allthekeeps/pull/3#discussion_r702938384), then can we handle this comment there as well?